angular.json

```diff
@@ -290,8 +313,13 @@
         "lint": {
           "builder": "@angular-devkit/build-angular:tslint",
           "options": {
-            "tsConfig": ["apps/okreads-e2e/tsconfig.e2e.json"],
-            "exclude": ["**/node_modules/**", "!apps/okreads-e2e/**/*"]
+            "tsConfig": [
+              "apps/okreads-e2e/tsconfig.e2e.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!apps/okreads-e2e/**/*"
+            ]
```

Comment:
All of this formatting churn is difficult to read. Did you consider setting your line length to fit?